analysis_options.yaml

```diff
@@ -0,0 +1,15 @@
+analyzer:
+  strong-mode: true
+# Upgrade lints to errors that will break Google3
+errors:
+  unused_element: error
+  unused_import: error
```

Comment:
I'm fairly certain this at least *doesn't* break Google3. I'm pretty sure we've had packages that ended up with unused imports lasting a long time without breaking anything.